README.md

```diff
@@ -73,6 +73,39 @@ It is based on the theme developed by the Home Office: https://github.com/UKHome
 
 For full documentation on Keycloak themes see: https://www.keycloak.org/docs/latest/server_development/index.html#_themes
 
+## GovUK Notify Service
+
+TDR Keycloak uses GovUK Notify as its default email sender provider.
+
+Each TDR environment has a separate GovUK Notify service defined:
+* TDR Intg - service for the integration environment
+* TDR Staging - service for the staging environment
+* TDR - service for the production environment
+
+The GovUK Notify Services require two secret values that are stored as AWS SSM pararmeters:
+* API Key: this is the key to access the GovUK Notify service
+* Template ID: this is the id of the GovUK Notify service email template
+
+Both these AWS SSM parameter *values* need to set manually as it is not programmatically possible to retrieve these from GovUK Notify
```

Comment:
Just to clarify - do they need to be set manually in the AWS parameter store? Not in Keycloak?